src/components/WebCLI/WebCLI.js

```diff
@@ -138,7 +97,7 @@ const WebCLI = ({ controllerWSHost, credentials, modelUUID, protocol }) => {
             type="text"
             name="command"
             ref={inputRef}
-            placeholder={placeholder}
+            placeholder="enter command"
```

Comment:
Should enter not be capitalised?

Reply:
Not as per the design.  I tried it as "Enter command" and it didn't look right in the prompt.